vehicle/simvehicle.cc

```diff
@@ -1859,23 +1859,30 @@ void vehicle_t::display_after(int xpos, int ypos, bool is_global) const
 	constexpr uint16 tooltip_text_size = 1024;
 	char tooltip_text[tooltip_text_size];
 	tooltip_text[0] = 0;
+	constexpr uint16 line_name_size = 512;
+	char line_name[line_name_size];
+	line_name[0] = 0;
+	constexpr uint16 states_text_size = 512;
+	char states_text[states_text_size];
+	states_text[0] = 0;
 	uint8 state = env_t::show_vehicle_states;
+	// color flag, 0 or 1->yellow,2->orange,3->red,4->green
+	uint8 color_flag = 0;
```

Comment:
置き換えた数字を使うよりも直接色を持ってしまった方がわかりやすそうです。
```suggestion
	PIXVAL tooltip_color_idx = 0;
```

Reply:
対応しました